### DISCUSSION TITLE
Broken links on main page?

### DISCUSSION THREAD

**Discussion by dkp** [executablebooks/meta#782] [Q&A]
I've just arrived at your https://jupyterbook.org/en/stable/intro.html page for the first time and found that prominently displayed 6 boxes at the top of the page all lead nowhere.  This seems like a mistake.  Either they should not be clickable or they should go somewhere.
See ![Built_with_Jupyter_Book](https://user-images.githubusercontent.com/965184/179101214-750f98bd-cb16-477d-9719-c77297274f70.png)

**Comment by chrisjsewell** [comment]
Yeh cheers, its adding `.md` to the end of all links, rather than `.html` 😒 

Double - checked https://sphinx-design.readthedocs.io/en/sbt-theme/ and they are all fine 😬 🤷

**Reply by dkp** [reply]
Ahh, then hopefully an easy fix. Thanks.

-Dianne

**Reply by choldgraf** [reply]
I believe this is because they are broken on "stable" but fixed on "latest". Can you confirm that they work on latest? It should be resolved when a new release of jupyter book occurs

**Comment by dkp** [comment]
You are correct, it is resolved on latest.  Thank you!
